rancher/rancher-ui/infrastructure/hosts/custom/index.md

```diff
@@ -40,9 +40,9 @@ http://<rancher-server-ip>:8080/v1/projects/1a5/scripts/<registrationToken>
 
 ### Security Groups/Firewalls 
 
-For any hosts that are added, ensure that any security groups or firewalls allow traffic. If these are not enabled, the Rancher functionality will be limited.
+For any hosts that are added, ensure that any security groups or firewalls allow traffic between the Rancher server and the Rancher agents, otherwise Rancher functionality will be limited.
 
-* From and to all other hosts on UDP ports `500` and `4500` (for IPsec networking)
+* From Rancher server to UDP ports `500` and `4500` (for IPsec networking) on Rancher agent hosts
 
```

Comment:
Thats what it says. In firewall lingo "From" means from the connecting server. So for example, on the host that the agent is running, you would add a firewall rule that says, "allow UDP **from** rancher-server-host to myself on port 500".